src/include/kdump/dialogs.rb

```diff
@@ -523,14 +523,19 @@ def tabs
     end
 
     def kdump_memory_widget
-      min = Kdump.memory_limits[:min_low].to_i
-      max = Kdump.memory_limits[:max_low].to_i
+      low_min = Kdump.memory_limits[:min_low].to_i
+      low_max = Kdump.memory_limits[:max_low].to_i
+      low_default = Kdump.memory_limits[:default_low].to_i
       low_label = if Kdump.high_memory_supported?
         _("Kdump &Low Memory [MiB]")
       else
         _("Kdump Memor&y [MiB]")
       end
-      low_label += " (#{min} - #{max})"
+      # TRANSLATORS: Each momory value is based on MiB
```

Comment:
typo momory and allso it is good that you mention it for translators, but it is not mentioned even in english not? It is already in label above, so I think it does not bring much value for translators.

Reply:
better mention to not modify that `%{}` part.